content/handbook/support/workflows/account_deletion_access_request_workflows.md

```
@@ -0,0 +1,327 @@
+
+- チケットが **データアクセス** リクエストに関するものであれば、ユーザーを GitLab [Privacy Center](https://privacy.gitlab.com) でリクエストを提出するようリダイレクトし、チケットを solved としてマークします。
+
+- チケットが **データポータビリティ** リクエストに関するものであれば、ユーザーにさらに明確化を求めます。ポータビリティリクエストはユーザーが GitLab から別のプラットフォームに移行したい場合のみ適用されます。その場合、GitLab はプロジェクトのインポートとエクスポート機能を介したセルフサービスの手順を提供します。移行先のプラットフォームは通常、ユーザーがあるプラットフォームから別のプラットフォームに移行またはポートするために必要な移行ドキュメントを提供します。
+
+- チケットが **データエクスポート** リクエストに関するものであれば、これはアクセスリクエストとして扱われます。ユーザーを GitLab [Privacy Center](https://privacy.gitlab.com) でリクエストを提出するようリダイレクトし、チケットを solved としてマークします。
+
+### **Stage 2:** リクエスト処理
+
+リクエストタイプに基づいて、以下から該当するワークフローを見つけてプライバシーリクエストを処理します。
+
+- [削除リクエスト](#deletion-requests)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)